feed-generator/Program.cs

```diff
@@ -0,0 +1,95 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace FeedGenerator
+{
+    using Dapr.Client;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Hosting;
+    using System;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// FeedGenerator - generates messages and publishes them using Dapr.
+    /// The main functionality is in StartMessageGeneratorAsync().
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main for FeedGenerator
+        /// </summary>
+        /// <param name="args">Arguments.</param>
+        public static void Main(string[] args)
+        {
+            IHost host = CreateHostBuilder(args).Build();
+
+            Task.Run(() => StartMessageGeneratorAsync());
+
+            host.Run();
+        }
+
+        /// <summary>
+        /// Creates WebHost Builder.
+        /// </summary>
+        /// <param name="args">Arguments.</param>
+        /// <returns>Returns IHostbuilder.</returns>
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
```

Comment:
Why does this need to be asp.net core application?

Reply:
It doesn't, but there's a comment at the top of this file about this content is here.